pytest_ci.ini

```diff
@@ -1,4 +1,4 @@
 [pytest]
 # run with full migrations, increased verbosity (-vvv) and enabled stdout (-s)
 DJANGO_SETTINGS_MODULE=pycon.dev-settings
-addopts = --cov-config .coveragerc --cov ./ --cov-report term --profile-svg --durations=5 -vvv -s
+addopts = --cov-config .coveragerc --cov ./ --cov-report term --durations=5 -vvv -s -n auto
```

Comment:
you don't disable the profilings, `--durations=5` is the key, and maybe could disable the code coverage?

Reply:
Coverage on CI can be helpful, so I'd keep that part, but profiling is definitely not needed.